src/content/updates/migration/designers.mdx

```diff
@@ -0,0 +1,158 @@
+---
+
+<AnchorLinks>
+
+- [IBM Design Language](#ibm-design-language)
+- [Design kit](#design-kit)
+- [Themes](#themes)
+- [Icons](#icons)
+- [Typography](#typography)
+- [Motion](#motion)
+- [Layout and grid](#layout-and-grid)
+- [Design migration strategy](#design-migration-strategy)
+<!-- - [Design FAQ](#design-faq) -->
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Resources
+
+<GridWrapper col_lg="8" flex="true" bleed="true">
+  <ClickableTile
+    title="Subscribe to Carbon Design Kit Library (Sketch Cloud)"
+    href="sketch://add-library/cloud/JaVzz"
+    type="resource">
+
+![](/images/sketch-icon.png)
+
+  </ClickableTile>
+
+  <ClickableTile
+    title="Download Carbon Design Kit Sketch file (GitHub)"
+    href="https://github.com/ibm/carbon-design-kit"
+    type="resource">
+
+![](/images/sketch-icon.png)
+</ClickableTile>
+
+<ClickableTile
+    title="Subscribe to IBM Design Language Library (Sketch Cloud)"
+    href="sketch://add-library/cloud/75VZZ"
+    type="resource">
+
+![](/images/sketch-icon.png)
+
+  </ClickableTile>
+
+</GridWrapper>
+
+## IBM Design Language
+
+Carbon X delivers the new <a href="https://www.ibm.com/design/language/" target="_blank">IBM Design Language</a> to product teams. It represents a complete visual refresh of the system, delivering the ethos of IBM Design. With tooling and  guidance on color, layout, typography, iconography, motion, patterns, and content, Carbon X is a modern, open-source framework for building digital products and experiences.
+
+## Design kit
+
+Carbon X includes a completely revised Sketch design kit, updated for the new IBM Design Language. All components in the kit have been aligned with the Design Language, as well as colors, icons, typography, motion, and grid elements. 
+
+Please note that upgrading to the v10 kit will not automatically change any files created in previous versions of the kit. In Sketch, the migration of existing v9 layouts will be a manual process for the designer.
+
+### What's changed from the v9 kit
+
+- New IBM Design Language styling
+- Improved kit structure
+- Updated text styles
+- Updated spacers
+- Icon updates
+- New color tokens
+- Updated elements page
+- New Getting Started page
+- Updated layer styles
+
+For more details on these changes to the design kit, see the What's New page in the <a href="https://github.com/ibm/carbon-design-kit" target="_blank">Design Kit Sketch file</a>.
+
+### Setting up the kit
+
+
+The preferred method for using the kit is via Sketch cloud subscription. Click the tile above to subscribe to the Carbon Design Kit as a library. Sketch cloud integration ensures that all updates to the design kit will be automatically pushed to you via notifications in the Sketch app. 
+
+Also download the Carbon Design Kit Sketch file from GitHub. Use this file for reference and guidance while working with the kit. If you are subscribed to the library, all changes in the master kit will be pushed to this file as well. 
```

Comment:
This isn't the process. Direct install of the Carbon kit from Sketchcloud , direct install of the IDL Kit from Sketchcloud, and direct install of the grid template are required. Downloading the sketchkit from the Github Repo is not _also_ a requirement but an alternative to Direct install of the Carbon kit from Sketchcloud.

Reply:
Additionally, if you DL the sketch library from gh to your computer, and even if you install it manually using the add a library function (Sketch Preferences : Libraries : Add Library…), it's orphaned - it never will get updates as we update the file on github. 

we _really_ need to remove the dl from gh method, or at least not give the impression they are equivalent.

Reply:
To add to this point, working with Sketch Cloud is a huge shift from how we worked with the kit in the past. I'd guess some of our users were not even aware it's possible now. It's an awesome thing! If we made it more of the point that you're not just migrating your design, you're getting a better way to work with our design system.

Reply:
Good feedback all around. Here's one thing that I'm unclear on still: Our design kit file is currently named with "beta" appended to the filename. Are we able to change that filename at the master level, so that someone who has already subscribed to the beta kit will see an automatic change to the "v10" kit? Or will those necessarily be handled as two separate libraries?

Reply:
Revised content to reflect the new preferred method for design kit. The netlify preview link reflects the new content.